aide_design/floc_model.py

```diff
@@ -54,22 +53,20 @@ def define_Precip(self, diameter, density, molecweight, alumMPM):
 # name, diameter in m, density in kg/m³, molecular weight in kg/mole
 Clay = Material('Clay', 7 * 10**-6, 2650, None)
 
-
 PACl = Chemical('PACl', (90 * u.nm).to(u.m).magnitude, 1138, 1.039,
                 'PACl', AluminumMPM=13)
 
-
 Alum = Chemical('Alum', (70 * u.nm).to(u.m).magnitude, 2420, 0.59921,
                 'AlOH3', AluminumMPM=2)
-Alum.define_Precip((70 * u.nm).to(u.m).magnitude, 2420, 0.078, 1)
 
+Alum.define_Precip((70 * u.nm).to(u.m).magnitude, 2420, 0.078, 1)
 
 HumicAcid = Chemical('Humic Acid', 72 * 10**-9, 1780, None, 'Humic Acid')
 
 
 ################### Necessary Constants ###################
 # Fractal diameter, based on data from Adachi.
```

Comment:
should be dimension, not diameter. can we more specifically reference Adachi,... see [here](http://online.liebertpub.com/doi/abs/10.1089/ees.2015.0302)